app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        self.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (f""
```

Comment:
<img width="348" alt="Screenshot 2023-10-11 at 17 09 29" src="https://github.com/mate-academy/py-herbivores-and-carnivores/assets/117740672/0fc179e3-1e9e-4b2c-b3de-a60975c0ec8c">